crates/circuits/sha256-air/src/sha256/columns.rs

```diff
@@ -0,0 +1,85 @@
+//! WARNING: the order of fields in the structs is important, do not change it
+
+use openvm_circuit_primitives::AlignedBorrow;
+
+use super::{
+    SHA256_HASH_WORDS, SHA256_ROUNDS_PER_ROW, SHA256_ROW_VAR_CNT, SHA256_WORD_BITS,
+    SHA256_WORD_U16S, SHA256_WORD_U8S,
+};
+
+/// the first 16 rows of every SHA256 block will be of type Sha256RoundCols and the last row will be of type Sha256DigestCols
```

Comment:
A comment somewhere around here about how a row in the air should either be transmuted to `Sha256RoundCols` vs `Sha256DigestCols` would be helpful